dijkstra/dijkstra.c

```diff
@@ -16,7 +16,7 @@
 int minimum_distance(const int min_distances[], const bool shortest_paths[]) {
 	int i;
 	int minimum = INFINITY, index;
-	for (int i = 0; i < VERTICES; ++i) {
+	for (i = 0; i < VERTICES; ++i) {
```

Comment:
I don't understand how we are not supposed to see an error here. If you have already declared `i` in the beginning, how can you declare it in the `for` loop again?